components/TradingCardImage.jsx

```diff
@@ -0,0 +1,35 @@
+export default function TradingCardImage({ imageUrl, hours }) {
+  const getModifier = (availableHours) => {
+    if (availableHours === 0) return 'gray';
+    if (availableHours >= 16) return 'green';
+    return 'blue';
+  };
+
+  const modifier = getModifier(hours);
```

Comment:
We talked about it while pairing, but for posterity, we decided that moving the `getModifier` into a utility function made more sense, since other places (like `<TradingCard>`) will be able to use the same logic.